### PR TITLE
Add support for release title customization via template

### LINKED ISSUE #27: Custom release title
Hey fregante,

I would like to request to add **custom title** option. Default it will be just tag name (`{tag}`). You could change it like follows:

```yaml
...
- name: Create release with custom title
  uses: fregante/release-with-changelog@v3
  with:
    token: ${{ secrets.GITHUB_TOKEN }}
    title: Version {tag}
```

That will produce release with title `Version 3.1.0` (when is tag `3.1.0`).

PS: Thanks for making very useful Github Action. It helps me a lot 😄

### PR DESCRIPTION
**Pull request by BelKed**
Resolves #27.

- [x] Add custom title option
- [x] Update README.md

### CODE REVIEW

**Comment by BelKed** [comment]
@fregante, please, could you review this pull request?

File: index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ async function run() {
 	try {
 		const {owner, repo} = context.repo;
 
+		let releaseTitle = core.getInput('title');
```
Comment (fregante) [inline index.js]:
```suggestion
		const releaseTitle = core.getInput('title');
```

And then below…

File: index.js [hunk 2]
```diff
@@ -41,11 +42,15 @@ async function run() {
...
 
+		// Replace {tag} with pushedTag
+		releaseTitle = releaseTitle.replace('{tag}', pushedTag);
+
 		// Create a release with markdown content in body
 		const octokit = getOctokit(core.getInput('token'));
 		const createReleaseResponse = await octokit.repos.createRelease({
 			repo,
 			owner,
+			name: releaseTitle,
```
Comment (fregante) [inline index.js]:
```suggestion
			name: releaseTitle.replace('{tag}', pushedTag),
```

**Review by fregante — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (BelKed) [reply index.js]: OK

File: index.js (hunk 2 above)
Comment (BelKed) [reply index.js]: OK

**Review by fregante — APPROVED** [review]

File: action.yml [hunk 3]
```diff
@@ -10,6 +10,9 @@ inputs:
   token:
     description: Personal access token (PAT) used to create releases
     required: true
+  title:
+    description: Custom tile
```
Comment (fregante) [inline action.yml]:
```suggestion
    description: Custom title for the release
```

**Comment by fregante** [comment]
I'll wait a few more days before releasing a new version. So if you want to open new PRs you can do so before then

**Comment by rm3195** [comment]
Hi @fregante! Any plans for releasing a new version with this change soon?
Thanks!

**Comment by fregante** [comment]
v3.3.0

**Pull request merged**